pdns/pdns_recursor.cc

```diff
@@ -1401,6 +1401,18 @@ static void startDoResolve(void *p)
     }
   sendit:;
 
+    if(g_useIncomingECS && haveEDNS && !sr.wasVariable()) {
+      //      cerr<<"Stuffing in a 0 scope because answer is static"<<endl;
+      EDNSSubnetOpts eo;
+      eo.source = dc->d_ednssubnet.source;
+      ComboAddress sa;
+      memset(&sa, 0, sizeof(sa));
```

Comment:
Please do `sa.reset()` instead, this will rightfully trigger a warning with `GCC` 8.1.0+